src/important-links.md

```diff
@@ -0,0 +1,16 @@
+# Ignore me
+
+This file is a collection of links that are not link-checked by anyone else,
+but we want them to work. For example, the stabilization guide link is often
+posted to tracking issues on GitHub, which might break later if the guide is
+changed.
+
+[Skip this file. Or read it; I'm a book, not a cop.](https://imgur.com/gallery/mSHi8)
+
+[link](https://forge.rust-lang.org/stabilization-guide.html)
+[link](https://forge.rust-lang.org/stabilization-guide.html#updating-documentation)
+[link](https://forge.rust-lang.org/stabilization-guide.html#documentation-prs)
```

Comment:
Uh, how does the link checker work? I doubt it will make sure the anchors exists here.

Reply:
You just beat me to posting: https://github.com/rust-lang/rustc-guide/pull/269#discussion_r249978904